apps/classroomio-com/src/routes/bootcamps/+page.svelte

```diff
@@ -0,0 +1,220 @@
+<script>
+  import PageSignupCTA from '$lib/PageSignupCTA/index.svelte';
+  import PageHeader from '$lib/PageHeader/PageHeader.svelte';
+
+  const benefitDetails = [
+    {
+      img: 'ai.svg',
+      title: 'AI-Powered Learning',
+      point_1: 'Create engaging courses with AI-assisted content generation.',
+      point_2: 'Automate grading and feedback processes, saving time and effort.',
+      point_3: 'Personalize learning experiences based on individual student needs.'
+    },
+    {
+      img: 'community.svg',
+      title: 'Foster a Thriving Community',
+      point_1:
+        'Create a supportive learning environment with built-in forums and interactive tools.',
+      point_2: 'Encourage collaboration and knowledge sharing among students.',
+      point_3: 'Build a strong community of learners and mentors.'
+    },
+    {
+      img: 'operations.svg',
+      title: 'Streamline Operations',
+      point_1: 'Manage your courses and students efficiently with our user-friendly platform.',
+      point_2: 'Issue custom certificates to validate student achievements.',
+      point_3: 'Provide a seamless learning experience for your students.'
+    }
+  ];
+
+  const featureDetails = [
+    {
+      img: 'certificate-i.svg',
+      img2: 'certificate.svg',
```

Comment:
Avoid using numbers in naming variables like `img2`. The reason is that numbers don't tell you what that variable is used for. You want to try to be as clear as possible when naming a variable.

In this case you should ask, what that `img2` actually represent? how can I capture that in a simple, yet clear name